src/components/pages/TeachersPage/Chatbox.css.tsx

```diff
@@ -6,16 +6,24 @@ const chatboxContainer = css`
 
 const chatboxTop = css`
   background: #f8e5e0;
-  border-radius: 10px 10px 10px 10px;
+  border-radius: 10px 10px 0 0;
```

Comment:
This change affects the bottom of the teachers page where `AllStudentChatsDisplayed` is used.  That component reuses the `ChatBox` component you have targeted. 

They now have a squared of bottom border radius.  Is this intentional?

Reply:
Thanks for catching this! The chats on the bottom should keep their round border radius as they look nicer that way.   But I don't think I'll solve it in this PR. Once this PR is accepted, before shipping, I will document this in an issue for any engineer to tackle later.  It'll be good to have an extra "low-priority" task for any new engineers in the future.

Reply:
Documented the need to round the corners on the bottom chats in Issue https://github.com/Frempco/web-client/issues/30